src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -473,14 +471,14 @@ protected function _bundleTranslatedFields($entity)
         }
 
         $results = $this->_findExistingTranslations($find);
-        $model = $this->config('conditions.model');
+        $alias = $this->_table->alias();
 
```

Comment:
I might be misunderstanding, that sounds like the model key is a hack to account for something we're missing. 

Are you saying that this won't work:

    $table = TableRegistry::get('Posts');
    $table->addBehavior('Translate', ...);
    $table->alias('AnythingAtAll');
    $untranslatedRecords = $table->find('all');

?

In 2.x the translate behavior uses the [model name](https://github.com/cakephp/cakephp/blob/master/lib/Cake/Model/Behavior/TranslateBehavior.php#L245) - rather than the model alias.

@markstory @lorenzo should we have a name getter for table objects? That's what structural behaviors would read, rather than the mutable alias. Right now it's [buried in the alias method](https://github.com/cakephp/cakephp/blob/3.0/src/ORM/Table.php#L354-L356)

In any event we have no relevant tests for the model key - which is in part why I was able to know nothing about it and have no tests fail.